src/panels/config/hardware/ha-config-hardware.ts

```diff
@@ -439,6 +442,16 @@ class HaConfigHardware extends SubscribeMixin(LitElement) {
                     ></ha-chart-base>
                   </div>
                 </ha-card>`
+            : this._hardwareComponentLoaded
```

Comment:
```suggestion
            : isComponentLoaded(this.hass, "hardware")
```

Reply:
It is not heavy, and also not each, only when it is not loaded yet, or when it is not loaded